packages/app/src/vis-packs/nexus/utils.ts

```diff
@@ -181,3 +182,9 @@ export function assertComplexSignal(
 ): asserts nxData is NxData<ComplexType> {
   assertComplexType(nxData.signalDataset);
 }
+
+export function assertScatterAxisParams(
+  params: AxisParams | undefined
+): asserts params is ScatterAxisParams {
+  assertDefined(params?.value);
+}
```

Comment:
This is not really a `nexus/utils` but it is used only in `NxScatterContainer` so I did not find a better place for it